### PR TITLE
Along with Connor's fixes, I think this is pretty close

### PR DESCRIPTION
**Pull request by JamesDunton**
There were some changes that I made to the statistics file along with the city menu and the validators for the city and province. When the client passes in the city and province for the location on the map, I'm not calling get specific city anymore. I did this so that we wouldn't get prompted again for a province name.

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -267,16 +264,17 @@ static string OneCityValidator(Statistics cityStats)
             string response;
             do
             {
-                Console.Write("Enter city name or type 'list' to display all cities: ");
+                Console.Write("\nEnter city name or type 'list' to display all cities: ");
                 response = Console.ReadLine();
+                response = response.Trim();
```
Comment (hunter-bennett28) [inline Program.cs]: this can be simplified just to response = Console.ReadLine().Trim();

File: Program.cs [hunk 2]
```diff
@@ -267,16 +264,17 @@ static string OneCityValidator(Statistics cityStats)
...
                 if (response.ToLower() == "list")
                 {
                     Console.Write("\n Enter the province name to see the cities located in that given province: ");
```
Comment (hunter-bennett28) [inline Program.cs:282]: I think we need to ALSO add an option to list the provinces for full hand hold mode. That can be done in a separate PR though

**Review by hunter-bennett28 — CHANGES_REQUESTED** [review]
This needs to be updated to merge in the changes from Connor's branch then we can merge this in and start doing more UI tailoring and error checking

File: Program.cs (hunk 2 above)
Comment (conninator2000) [reply Program.cs:282]: We could just display the list of provinces then say "pick a province to display cities"

File: Program.cs [hunk 3]
```diff
@@ -326,19 +325,29 @@ static string[] TwoCityValidator(Statistics cityStats)
             StringBuilder str = new StringBuilder("");
             do
             {
-                Console.Write("Enter two cities split by a comma (ex: London, Chatham), or type 'list' to see a list of cities: ");
+                Console.Write("\nEnter two cities split by a comma (ex: London, Victoria), or type 'list' to see a list of cities: ");
                 response = Console.ReadLine();
+                response = response.Trim();
```
Comment (conninator2000) [inline Program.cs]: Can be shortened by doing the ReadLine().Trim()

**Review by conninator2000 — COMMENTED** [review]
Looks ok, looking to get this merged so I can test out the program some more

File: Program.cs (hunk 2 above)
Comment (hunter-bennett28) [reply Program.cs:282]: Yeah I agree

**Pull request merged**